docker/init_scripts/init

```diff
@@ -115,19 +115,43 @@ watchdog_process_pid() {
 	fi
 }
 
+stop_process_pid() {
+	PROCESS=$1
+  if [[ -f "/tmp/${PROCESS}.pid" ]]; then
+    PID=$(cat "/tmp/${PROCESS}.pid") || true
+		if [[ -d "/proc/${PID}" ]]; then
+		  info_log "stopping ${PROCESS}"
+		  kill "${PID}" || true
+		  # wait for process exit
+		  while [ -e "/proc/${PID}" ]; do sleep 0.1; done
```

Comment:
nit: Seems indentation is broken within this function.

Reply:
fixed, converted to tabs